hack/common.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+get_image_tag() {
+    local current_commit today
+    current_commit="$(git rev-parse HEAD)"
+    today="$(date +%Y%m%d)"
+    echo "v${today}-${current_commit:0:7}"
+}
+
+SSP_DIR="$(readlink -f $(dirname $0)/../)"
+WORK_DIR="/go/src/github.com/kubevirt/ssp-operator"
+REGISTRY=${REGISTRY:-quay.io/vparekh}
+REPOSITORY=${REPOSITORY:-ssp-builder}
+TAG=$(get_image_tag)
+BUILD_TAG="${REGISTRY}/${REPOSITORY}:${TAG}"
```

Comment:
This name is also confusing, I think we should prefix all test-related variables with `TEST_*` or something